pkg/sql/privilege/privilege.go

```diff
@@ -102,7 +103,7 @@ var (
 	// certain privileges unavailable after upgrade migration.
 	// Note that "CREATE, INSERT, DELETE, ZONECONFIG" are no-op privileges on sequences.
 	SequencePrivileges = List{ALL, USAGE, SELECT, UPDATE, CREATE, DROP, INSERT, DELETE, ZONECONFIG}
-	SystemPrivileges   = List{ALL, MODIFYCLUSTERSETTING}
+	SystemPrivileges   = List{ALL, MODIFYCLUSTERSETTING, CREATEEXTERNALCONNECTION}
```

Comment:
should this be called "manageexternalconnection" or even .. just "externalconnection"?

Reply:
`manage` might be a little confusing cause my understand is that we are going to add more fine-grained `DROP` and `ALTER` privileges for the external connection object. This system privilege will only guard one operation which is the creation of the object. We already have `CREATEDB` and `CREATEROLE` role options which I believe will soon become system privs, so probably worth being consistent with those?

Reply:
Sure; I was just wondering if we'll have just 1 priv (in which case renaming might make sense), or the plan is to add more.  Sounds like the plan is to add more privileges; so name is fine.

Reply:
I find the word `CREATEEXTERNALCONNECTION` to be hard to read - would it be terribly unconventional to do `CREATE_EXTERNAL_CONNECTION` instead, or something? Just to split apart those two `E`s...

Reply:
I defer to @RichardJCai or @rafiss on conventions here, I don't have strong opinions as long as it's clear what the privilege is for.

Reply:
does this mean there is no way to "drop" an external connection? i'd think the name EXTERNALCONNECTION makes sense. also, nothing here is case-sensitive so `GRANT SYSTEM ExternalConnection TO ...` is just as valid of a sql statement.

Reply:
There is a `DROP EXTERNAL CONNECTION` but I was going to gate that behind the user having a `DROP` privilege on that object rather than a system privilege. I'm going to use the synthetic privileges work Richard is introducing for virtual tables to add `DROP`, `ALTER` and `USAGE` privileges to External Connections. There are some more details in the permission model sections of this RFC https://github.com/cockroachdb/cockroach/pull/84209/files#diff-f495d13366596d9950932e4583ac3cc170ac1a1b29c62e4c04e63602b3fd49abR294.

`EXTERNALCONNECTION` sounds good to me.

Reply:
Changed to `EXTERNALCONNECTION`!